source/MaterialXView/Viewer.cpp

```diff
@@ -208,9 +208,20 @@ Viewer::Viewer(const mx::StringVec& libraryFolders,
     const mx::MeshList& meshes = _envGeometryHandler->getMeshes();
     if (!meshes.empty())
     {
+        // Invert u
+        mx::MeshPtr mesh = meshes[0];
+        mx::MeshStreamPtr stream = mesh->getStream(mx::MeshStream::TEXCOORD_ATTRIBUTE, 0);
+        mx::MeshFloatBuffer &buffer = stream->getData();
+        size_t stride = stream->getStride();
+        for (size_t i = 0; i < buffer.size() / stride; i++)
```

Comment:
Instead of trying to perform 1-V in the shader it is done here. Load time is a bit slower but avoid re-computation at shader time. Also did not seem easy to set up an invert on V only for texcoord.

Reply:
If we wanted to keep this as a shader computation, one option would be to use a scalar "invert" node (with the channels attribute selecting the U-coordinate) followed by a vector2 "combine" node (with the channels attribute selecting the V-coordinate).  Let me know how this approach sounds to you.